pkg/fqdn/cell/cell.go

```diff
@@ -22,5 +24,11 @@ var Cell = cell.Module(
 	// The FQDN bootstrap logic
 	bootstrap.Cell,
 
+	// The FQDN Message handler
+	messagehandler.Cell,
+
+	// GRPC server for the standalone DNS proxy
```

Comment:
I would expand this to say what data this pushes over to the standalone DNS proxy. My thinking with comments inside `cell.Module` is that you should be able to skim the comments to understand what each piece does and how data flows through the system. E.g. think about what you would like to see when you are here the first time and trying to puzzle what are all these building blocks doing. And yeah the existing comments we have aren't really setting a good example... If I would have a magic wand I'd wave it and all the `cell.Module` calls would get an ascii diagram comment on top of them showing how everything ties together and why :).

Reply:
Added the comments